### PR TITLE
fix: nexe binaries location

### PR DESCRIPTION
**Pull request by royra**
update location for nexe binaries after old location stopped working

### CODE REVIEW

File: packages/cli/package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "jest": "^24.9.0",
     "jest-junit": "^6.4.0",
     "memory-streams": "^0.1.3",
-    "nexe": "4.0.0-beta.4",
+    "nexe": "4.0.0-beta.6",
```
Comment (noamhammer) [inline packages/cli/package.json:82]: why this upgrade is needed?

File: packages/cli/package_native.js [hunk 2]
```diff
@@ -92,6 +97,7 @@ const doNexe = (input) => new Promise((resolve, reject) => {
     }
 
     console.log('Running nexe for platform %o', config.target)
+    console.dir(config)
```
Comment (noamhammer) [inline packages/cli/package_native.js:100]: for debugging purposes? you want to keep it?

File: packages/cli/package_native.js [hunk 3]
```diff
@@ -43,10 +44,14 @@ const BASE_NEXE_CONFIG = {
...
-  .map(([platform, platformOpts = {}]) => ({
-    output: `${path.join(TARGET_DIR, platform, TARGET_FILE_BASENAME)}${platformOpts.ext || ''}`,
-    target: { platform, arch: TARGET_ARCH, version: TARGET_NODE_VERSION },
-  }))
+  .map(([platform, platformOpts = {}]) => {
+    const target = { platform, arch: TARGET_ARCH, version: TARGET_NODE_VERSION }
+    return {
+      output: `${path.join(TARGET_DIR, platform, TARGET_FILE_BASENAME)}${platformOpts.ext || ''}`,
+      target,
+      asset: `https://github.com/nexe/nexe/releases/download/v3.3.3/${getTarget(target).toString()}`,
```
Comment (noamhammer) [inline packages/cli/package_native.js:52]: are we using v3.3.3 or v.4.0.0-beta*? what am I missing?

File: packages/cli/package_native.js (hunk 2 above)
Comment (royra) [reply packages/cli/package_native.js:100]: removed

File: packages/cli/package_native.js (hunk 3 above)
Comment (royra) [reply packages/cli/package_native.js:52]: we're using v4-beta, the downloads are from v3.3.3 (they were from v3.0.0 earlier, but that version was removed).

File: packages/cli/package.json (hunk 1 above)
Comment (royra) [reply packages/cli/package.json:82]: to stay as close as possible to the master branch.

**Pull request merged**